scripts/deploy/values/users-chis-ug.yaml

```
@@ -0,0 +1,25 @@
+cht-user-management:
+  image:
+    tag: 0.0.1
```

Bumping this to latest so we can merge this PR and have it be clearer how to push to prod for monday deadline: 

```suggestion
    tag: 1.0.0
```